scrapy/commands/parse.py

```diff
@@ -233,6 +238,22 @@ def process_spider_arguments(self, opts):
             opts.spargs = arglist_to_dict(opts.spargs)
         except ValueError:
             raise UsageError("Invalid -a value, use -a NAME=VALUE", print_help=False)
+        if opts.output:
```

Comment:
Please use [`scrapy.utils.conf.feed_process_params_from_cli`](https://github.com/scrapy/scrapy/blob/8845773d44329194ee73fa4985a94e768bf664e7/scrapy/utils/conf.py#L127-L172) to handle this. `FEED_URI` and `FEED_FORMAT` are deprecated in #3858 (to be released in 2.1)

Reply:
Okay @elacuesta. Thanks a lot for mentioning 🙂